packages/api/src/brevo-contact/brevo-contact.resolver.ts

```diff
@@ -93,10 +93,11 @@ export function createBrevoContactResolver({
         }
 
         @Query(() => PaginatedBrevoContacts)
-        async manuallyAssignedBrevoContacts(
-            @Args() { offset, limit, email, targetGroupId }: ManuallyAssignedBrevoContactsArgs,
+        async assignedBrevoContacts(
+            @Args() { offset, limit, email, targetGroupId, onlyManuallyAssigned }: AssignedBrevoContactsArgs,
```

Comment:
I just realised that there were already 2 different queries before for manually assigned and all brevo contacts in a target group. 

the brevo contacts query has a targetGroupId argument. So you can get all assigned contacts with this query 
 ```
 brevoContacts(targetGroupId: $targetGroupId, offset: $offset, limit: $limit, scope: $scope) {  
 ```
 and the `manuallyAssignedBrevoContacts` should just be used for the manually assigned contacts. 
 
 Should have remembered this sooner, sorry!